ansible/roles/etcd/tasks/install.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Force etcd_source_type to package manager 
+  set_fact:
+    etcd_source_type: "package-manager"
+  when: not is_atomic and ansible_distribution != "Ubuntu" and not is_coreos
+
+- name: Install etcd
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+        name: etcd
+        state: latest
+  when: etcd_source_type == "package-manager" 
+ 
+- name: Set the etcd_modified fact to true 
+  set_fact:
+    etcd_modified: true
+  when: etcd_source_type == "package-manager" 
```

Comment:
It makes sense to set the ``etcd_modified: true`` only for ``package-manager`` as installing etcd from github just downloads the binary without any service file. So really nothing to [re]start.